### ISSUE TITLE
ion-action-sheet-method-create-parameters-renamed rule threw an error

### ISSUE THREAD

**Issue by mburger81** [ionic-team/v4-migration-tslint#22]
Hello,
we already are working on migration our app using alpha 7, but we would love to test your migration tool. We installed last new version `1.2` (version before didn't work at all for us) and have manly two errors

1)
```
The 'ion-action-sheet-method-create-parameters-renamed' rule threw an error in '/abc/ionic4/web-client-ionic/src/app/services/message-controller/mesage-controller.service.ts':
TypeError: node.arguments[0].properties is not iterable
    at ActionSheetMethodCreateParametersRenamedWalker.visitCallExpression (/abc/ionic4/web-client-ionic/node_modules/@ionic/v4-migration-tslint/rules/ionActionSheetMethodCreateParametersRenamedRule.js:31:52)
    at ActionSheetMethodCreateParametersRenamedWalker.SyntaxWalker.visitNode (/abc/ionic4/web-client-ionic/node_modules/tslint/lib/language/walker/syntaxWalker.js:305:22)
    at /abc/ionic4/web-client-ionic/node_modules/tslint/lib/language/walker/syntaxWalker.js:535:63
    at visitNode (/abc/ionic4/web-client-ionic/node_modules/typescript/lib/typescript.js:13263:24)
    at Object.forEachChild (/abc/ionic4/web-client-ionic/node_modules/typescript/lib/typescript.js:13453:24)
    at ActionSheetMethodCreateParametersRenamedWalker.SyntaxWalker.walkChildren (/abc/ionic4/web-client-ionic/node_modules/tslint/lib/language/walker/syntaxWalker.js:535:12)
    at ActionSheetMethodCreateParametersRenamedWalker.SyntaxWalker.visitNode (/abc/ionic4/web-client-ionic/node_modules/tslint/lib/language/walker/syntaxWalker.js:530:22)
    at /abc/ionic4/web-client-ionic/node_modules/tslint/lib/language/walker/syntaxWalker.js:535:63
    at visitNode (/abc/ionic4/web-client-ionic/node_modules/typescript/lib/typescript.js:13263:24)
    at Object.forEachChild (/abc/ionic4/web-client-ionic/node_modules/typescript/lib/typescript.js:13349:21)
The 'ion-alert-method-create-parameters-renamed' rule threw an error in '/abc/ionic4/web-client-ionic/src/app/services/message-controller/mesage-controller.service.ts'
```:


2)
```
Could not find implementations for the following rules specified in the configuration:
    ion-item-option-is-now-an-element
Try upgrading TSLint and/or ensuring that you have all necessary custom rules installed.
If TSLint was recently upgraded, you may have old rules configured which need to be cleaned up.

```

Thx a lot for your supportin us on migration to v4 apps, there are a lot of to do and a tool like this could help many of us.

**Comment by dwieeb** [comment]
The first issue is because we're expecting a very specific syntax for the action sheet and alert controller `create()` methods. Could you share a snippet of code where that `create()` method is being used so we can see what you're doing? The syntax we're expecting is in the tests: https://github.com/ionic-team/v4-migration-tslint/blob/develop/test/ionActionSheetMethodCreateParametersRenamed.spec.ts

The second issue was just a documentation issue. The rule was checked but not available. Delete the `"ion-item-option-is-now-an-element": true,` line in your `ionic-migration.json` file to avoid the issue. I believe https://github.com/ionic-team/v4-migration-tslint/issues/11 will make this tool super easy to use--the `ionic-migration.json` file wouldn't even be needed. 

Please feel free to make two separate issues next time so there's less context switching.

**Comment by mburger81** [comment]
Hi, the problem seems coming from `Observable.create`, this snippet runs into error

```
import { Injectable } from '@angular/core';
import { Observable } from 'rxjs/Observable';

@Injectable({
    providedIn: 'root'
})
export class AppStatusService {

    constructor () {
        this.abc();
    }

    private abc(): Observable<any> {

        return Observable.create((observer) => {
            return observer.complete();
        });
    }
}
```

**Comment by dwieeb** [comment]
@mburger81 Thanks. That is fixed in https://github.com/ionic-team/v4-migration-tslint/pull/23

I will release a new version now.

**Comment by mburger81** [comment]
Bug seems to be closed with `v1.3` :+1:
